lib/IDE/Refactoring.cpp

```diff
@@ -2602,6 +2602,111 @@ bool RefactoringActionLocalizeString::performChange() {
   return false;
 }
 
+static void generateMemberwiseInit(SourceEditConsumer &EditConsumer,
+                            SourceManager &SM,
+                            SmallVectorImpl<std::string>& memberNameVector,
+                            SmallVectorImpl<std::string>& memberTypeVector,
+                            SourceLoc targetLocation) {
+  
+  assert(!memberTypeVector.empty());
+  assert(memberTypeVector.size() == memberNameVector.size());
+  
+  EditConsumer.accept(SM, targetLocation, "\ninternal init(");
+  
+  for (size_t i = 0; i < memberTypeVector.size(); i++) {
```

Comment:
Change this line to `for (size_t i = 0, n = memberTypeVector.size(); i < n; i++) {`. We don't need to call `size()` every time we evaluate the loop condition.